H/HiGHS/highs_common.jl

```diff
@@ -11,11 +11,15 @@ sources = [
         "https://github.com/ERGO-Code/HiGHS.git",
         "7a3f716bb2ad2df67525db8414ae48f5226dedd6",
     ),
-    DirectorySource("./bundled"),
+    DirectorySource(joinpath(@__DIR__, "bundled")),
 ]
 
-# Bash recipe for building across all platforms
-script = raw"""
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms())
+
+function build_script(; shared_libs::String)
+    return "BUILD_SHARED=$(shared_libs)\n" * raw"""
 cd $WORKSPACE/srcdir
 if [[ "${target}" == *86*-linux-musl* ]]; then
```

Comment:
Side note, since you use GCC 6, since shouldn't be necessary.

Reply:
You're a continual fount of wisdom.